keyboards/2_milk/config.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1209
+#define PRODUCT_ID      0xB195
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    rionlion100
+#define PRODUCT         2% Milk
+#define DESCRIPTION     A milk themed 2% Keyboard
+
+/* key matrix size */
+#define MATRIX_ROWS 2
```

Comment:
You may actually want to use the `DIRECT_PINS` option, here:

https://docs.qmk.fm/#/hardware_avr?id=direct-pin-matrix

If the keyboard and it's matrix is already designed, then ignore.  If not ... this is a simpler option.

Reply:
Thanks didn't know about Direct_pins will do

Reply:
I figured as much.  That's why I wanted to let you know. :)

Reply:
fixed 0297d7b